src/inmanta/ast/statements/call.py

```diff
@@ -75,7 +97,7 @@ def execute_direct(self, requires):
             raise RuntimeException("emits_statements functions are not allowed in direct execution")
         else:
             try:
-                return function(*arguments)
+                return function(*arguments, **kwargs)
```

Comment:
this can result in a TypeError if one of the kwargs is also passed positionally.

```
def a(x):
...   pass
... 
>>> a(x="a", **{"x":"x"})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: a() got multiple values for keyword argument 'x'
```

This exception should also result in clean RuntimeException, not a WrappingRuntimeException